rt/rs/extensions/rx2/src/main/java/org/apache/cxf/jaxrs/rx2/client/ObservableRxInvokerImpl.java

```diff
@@ -147,34 +148,90 @@ public ObservableRxInvokerImpl(WebClient wc, ExecutorService ex) {
 
     @Override
     public <T> Observable<T> method(String name, Entity<?> entity, Class<T> responseType) {
+        
+        Observable<T> observable = Observable.create(new ObservableOnSubscribe<T>() {
```

Comment:
If you think previous comments for `Flowable` worth the change, we could replicate it for `Observable` as well.

Reply:
Agree.  Thanks for the comments.  Done.